pkg/volume/glusterfs/glusterfs_test.go

```diff
@@ -160,7 +160,7 @@ func TestPluginPersistentVolume(t *testing.T) {
 		},
 		Spec: v1.PersistentVolumeSpec{
 			PersistentVolumeSource: v1.PersistentVolumeSource{
-				Glusterfs: &v1.GlusterfsVolumeSource{EndpointsName: "ep", Path: "vol", ReadOnly: false},
+				Glusterfs: &v1.GlusterfsPersistentVolumeSource{EndpointsName: "ep", Path: "vol", ReadOnly: true},
```

Comment:
why did readonly flip here?

Reply:
move the readonly change into its own PR. need to think through the implications more carefully, and want to keep this change purely mechanical

Reply:
ReadOnly should remain true since the readonly behavior change was moved out of this PR

Reply:
Done.